### PR TITLE
Add cuspatial

### PR DESCRIPTION
**Pull request by mluukkainen**
- Add cuspatial to base and devel images. Runtime image inherits from base
- Move RAPIDS_CONDA_VERSION_SPEC to 0.10* due to cuspatial dependencies
- Regenerate Dockerfiles

### CODE REVIEW

File: generatedDockerfiles/Dockerfile.centos7-devel [hunk 1]
```diff
@@ -212,6 +217,7 @@ CMD [ "/bin/bash" ]
 RUN source activate rapids \
     && env \
     && conda list \
+    && conda uninstall -y blas openblas \
```
Comment (mluukkainen) [inline generatedDockerfiles/Dockerfile.centos7-devel]: Blas is coming from previous commit. It looks like dockerfiles were not regenerated for blas patch and they show up now

File: dockerArgs [hunk 2]
```diff
@@ -15,7 +15,7 @@ ARG IPYTHON_VERSION=7.3*
 
 # base/runtime install specs
 ARG DASK_XGBOOST_CONDA_VERSION_SPEC=0.2*
-ARG RAPIDS_CONDA_VERSION_SPEC=0.9*
+ARG RAPIDS_CONDA_VERSION_SPEC=0.10*
```
Comment (mluukkainen) [inline dockerArgs]: cuspatial was not available for 0.09 so I had to bump version to 0.10* to make adding cuspatial possible

File: generatedDockerfiles/Dockerfile.centos7-devel [hunk 3]
```diff
@@ -180,6 +180,11 @@ RUN cd ${RAPIDS_DIR}/rmm && \
 RUN cd ${RAPIDS_DIR}/cudf && \
     source activate rapids && \
     ./build.sh
+RUN cd ${RAPIDS_DIR}/cuspatial && \
+    source activate rapids && \
+    export CUSPATIAL_HOME="$PWD" && \
```
Comment (mluukkainen) [inline generatedDockerfiles/Dockerfile.centos7-devel]: Should I do export here or maybe use the generated build script similar to xgboost?

Comment (rlratzel) [reply generatedDockerfiles/Dockerfile.centos7-devel]: This is fine for now. Can you also put a FIXME comment saying that we shouldn't need to set these vars long-term?

File: generatedDockerfiles/Dockerfile.centos7-devel (hunk 1 above)
Comment (rlratzel) [reply generatedDockerfiles/Dockerfile.centos7-devel]: Hopefully they'll get updated. Can you put a FIXME comment saying this should also be temporary?

File: dockerArgs (hunk 2 above)
Comment (rlratzel) [reply dockerArgs]: This seems correct to me, and probably something we needed to do earlier anyway.

File: generatedDockerfiles/Dockerfile.centos7-devel (hunk 3 above)
Comment (mluukkainen) [reply generatedDockerfiles/Dockerfile.centos7-devel]: Fixed

File: generatedDockerfiles/Dockerfile.centos7-devel (hunk 1 above)
Comment (mluukkainen) [reply generatedDockerfiles/Dockerfile.centos7-devel]: Fixed

File: generatedDockerfiles/Dockerfile.centos7-devel [hunk 4]
```diff
@@ -147,7 +147,7 @@ RUN source activate rapids \
     && conda remove --force \
          rmm librmm \
          nvstrings libnvstrings \
-         cudf dask-cudf libcudf \
+         cudf cuspatial dask-cudf libcudf \
```
[the remark this answers is not in the document]
Comment (mluukkainen) [reply generatedDockerfiles/Dockerfile.centos7-devel]: Fixed

**Pull request merged**